### PR TITLE
fix acceptance tests, substituting versionId with versionName

### PR DESCRIPTION
**Pull request by angeltabar**
fix url composition on acceptance tests. This way avoids errors with double bar or no bar, and other common errors in url composition

### CODE REVIEW

File: tests/acceptance/main_test.go [hunk 1]
```diff
@@ -22,7 +23,11 @@ func TestMain(m *testing.M) {
 	flag.Parse()
 	opts.Paths = flag.Args()
 
-	testRunner := integrationtests.NewTestRunner()
+	testRunner, err := integrationtests.NewTestRunner()
+	if err != nil {
+		log.Println("The unittests URL is invalid")
```
Comment (angelsanchez) [inline tests/acceptance/main_test.go:28]: at this point, the cause of the err could be an invalid URL or another thing.
The program should not assume that invalid URL is the cause.

File: tests/acceptance/main_test.go [hunk 2]
```diff
@@ -22,7 +23,11 @@ func TestMain(m *testing.M) {
...
+		os.Exit(1)
```
Comment (angelsanchez) [inline tests/acceptance/main_test.go:29]:
```
log.Println(...)
os.Exit(1)
```
could be simpliflied using `log.Fatal(...)`

File: tests/acceptance/main_test.go (hunk 1 above)
Comment (angelsanchez) [inline tests/acceptance/main_test.go:28]: why "unittests"?

File: tests/acceptance/graphql/graphql.go [hunk 3]
```diff
@@ -21,15 +22,21 @@ type GQManager struct {
...
 
-func NewGQManager(cfg config.Config) *GQManager {
-	graphQLURL := fmt.Sprintf("%s/graphql", cfg.APIBaseURL)
+func NewGQManager(cfg config.Config) (*GQManager, error) {
+	u, err := url.Parse(cfg.APIBaseURL)
+	if err != nil {
+		return nil, err
+	}
+	u.Path = path.Join(u.Path, "/graphql")
+	graphQLURL := u.String()
```
Comment (angelsanchez) [inline tests/acceptance/graphql/graphql.go:31]: This code is the same that `tests/acceptance/graphql/auth.go`:
```	u, err := url.Parse(g.cfg.APIBaseURL)
	if err != nil {
		return err
	}
	u.Path = path.Join(u.Path, "/api/v1/auth/token/signin")
	signInURL := u.String()```

Consider create a function to remove the code duplication. Eg `getAPIURL(path string) string`

File: tests/acceptance/main_test.go (hunk 1 above)
Comment (angeltabar) [reply tests/acceptance/main_test.go]: That's true. Fixed

File: tests/acceptance/main_test.go (hunk 2 above)
Comment (angeltabar) [reply tests/acceptance/main_test.go]: Changed

File: tests/acceptance/main_test.go (hunk 1 above)
Comment (angeltabar) [reply tests/acceptance/main_test.go]: Oooops! changed :)

File: tests/acceptance/graphql/graphql.go (hunk 3 above)
Comment (angeltabar) [reply tests/acceptance/graphql/graphql.go]: Ok, function created

File: tests/acceptance/main_test.go [hunk 4]
```diff
@@ -22,7 +23,10 @@ func TestMain(m *testing.M) {
 	flag.Parse()
 	opts.Paths = flag.Args()
 
-	testRunner := integrationtests.NewTestRunner()
+	testRunner, err := integrationtests.NewTestRunner()
+	if err != nil {
+		log.Fatalf("unexpected error with integration tests runner: %p \n", err)
```
Comment (angelsanchez) [inline tests/acceptance/main_test.go]:
```suggestion
		log.Fatalf("unexpected error with integration tests runner: %s", err)
```

**Review by angelsanchez — APPROVED** [review]

**Pull request merged**